scripts/zkai_attention_kv_two_head_bounded_weighted_native_gate.py

```
@@ -0,0 +1,519 @@
+        "final_kv_cache_commitment": input_payload["final_kv_cache_commitment"],
+        "outputs_commitment": input_payload["outputs_commitment"],
+    }
+
+
+def mutation_cases_for(payload: dict[str, Any]) -> list[dict[str, Any]]:
+    cases = []
+    for name in EXPECTED_MUTATION_NAMES:
+        mutated = mutate_payload(payload, name)
+        try:
+            validate_payload(mutated, allow_missing_mutation_summary=True)
+        except Exception as err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)